agir/donations/admin/filters.py

```diff
@@ -0,0 +1,95 @@
+from django.urls import reverse
+from django.utils.html import format_html
+
+from agir.groups.models import SupportGroup
+from agir.lib.admin.autocomplete_filter import AutocompleteRelatedModelFilter
+from agir.lib.admin.form_fields import AutocompleteSelectModel
+from agir.people.models import Person
+
+
+class MonthlyAllocationSubscriptionPersonFilter(AutocompleteRelatedModelFilter):
+    title = "Personne"
+    parameter_name = "subscription_person"
+
+    def get_rendered_widget(self):
+        widget = AutocompleteSelectModel(
+            Person,
+            self.model_admin.admin_site,
+        )
+        FieldClass = self.get_form_field()
+        field = FieldClass(
+            queryset=self.get_queryset_for_field(),
+            widget=widget,
+            required=False,
+        )
+        self._add_media(self.model_admin, widget)
+
+        attrs = self.widget_attrs.copy()
+        attrs["id"] = "id-%s-autocomplete-filter" % self.field_name
+        attrs["class"] = f'{attrs.get("class", "")} select-filter'.strip()
+
+        return field.widget.render(
+            name=self.parameter_name,
+            value=self.used_parameters.get(self.parameter_name, ""),
+            attrs=attrs,
+        ) + format_html(
+            '<a style="margin-top: 5px" href="{}">Gérer les personnes</a>',
+            reverse("admin:people_person_changelist"),
+        )
+
+    def get_queryset_for_field(self):
+        return Person.objects.all()
+
+    def queryset(self, request, queryset):
+        if self.value():
+            try:
+                person = Person.objects.get(pk=self.value())
+                return queryset.filter(subscription__person_id=person.pk)
+            except Person.DoesNotExist:
+                pass
+
+        return queryset
+
+
+class MonthlyAllocationGroupFilter(AutocompleteRelatedModelFilter):
+    title = "group"
+    parameter_name = "group"
+
+    def get_rendered_widget(self):
+        widget = AutocompleteSelectModel(
+            SupportGroup,
+            self.model_admin.admin_site,
+        )
+        FieldClass = self.get_form_field()
+        field = FieldClass(
+            queryset=self.get_queryset_for_field(),
```

Comment:
Pourquoi ne pas passer directement la queryset si elle n'est pas utilisée ailleurs ?
```suggestion
            queryset=SupportGroup.objects.all(),
```